src/jit/codegenlegacy.cpp

```diff
@@ -22410,10 +22367,6 @@ void                CodeGen::genPInvokeCallEpilog(LclVarDsc *  frameListRoot,
     {
         if (retVal == RBM_INTRET || retVal == RBM_LNGRET)
         {
-#ifdef _TARGET_AMD64_
-            retTmp = compiler->tmpGetTemp(TYP_LONG);
-            inst_ST_RV(INS_mov, retTmp, 0, REG_INTRET, TYP_LONG);
-#elif defined(_TARGET_X86_)
```

Comment:
OK, I see it now. The diff view seems to be omitting the portion of the file with the `#ifdef _TARGET_ARM_`.